quantum/split_common/matrix.c

```diff
@@ -30,8 +30,16 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "pro_micro.h"
 #include "config.h"
 #include "timer.h"
+#include "split_flags.h"
 
-#ifdef USE_I2C
+#ifdef RGBLIGHT_ENABLE
+#   include "rgblight.h"
+#endif
+#ifdef BACKLIGHT_ENABLE
+#   include "backlight.h"
+#endif
+
+#if defined(USE_I2C) || defined(EH)
```

Comment:
There are several occurrences of `defined(USE_I2C) || defined(EH)` checks under the new `quantum/split_common` -- I assme that `split_common` shouldn't be looking for keyboard specific defines, so eh should just ensure that it defines USE_I2C or the EH define should be renamed to something semantic and keyboard agnostic?

Reply:
Ah, this was a holdover for merging lets_split_eh into lets_split. Since lets_split defines Serial/I2C at a keymap level and the "EH" version can only be I2C I needed to add it's own define. Otherwise you wouldn't be able to compile other keymaps with it. Since I'm no-longer merging into lets_split (lets_split_eh is it's own keyboard folder for now) I'll look into removing it